pennylane/gradients/param_shift_hessian.py

```diff
@@ -162,29 +193,31 @@ def processing_fn(results):
         out_dim = qml.math.shape(results)[1:]
         # The desired shape of the Hessian is:
         #       (QNode output dimensions, # trainable gate args, # trainable gate args),
-        # but first we accumulate all elements into a list, since no array assingment is possible.
+        # but first we accumulate all elements into a list, since no array assignment is possible.
         hessian = []
         # Keep track of tape results already consumed.
-        start = 1 if unshifted_coeffs and f0 is None else 0
+        start = int(bool(unshifted_coeffs) and f0 is None)
```

Comment:
Is this change functionally different? I feel like the previous formulation was slightly more readable :)

Reply:
Mhm. No it is not, and I actually agree it was more readable before... This might be a leftover from an attempt to restructure the tape list a little, which I didn't end up keeping. Will revert :)